src/App.js

```diff
@@ -29,6 +30,18 @@ function App() {
     setIsModalOpen(true);
   };
 
+  // Load Airtable data once on each app render
+  useEffect(() => {
+    const loadData = async () => {
```

Comment:
Code looks okay here, but would you be implementing ways to send out multiple API requests for paginated stuff? Ideally these requests should happen asynchronously, when the page loads, and not on each render.

To correct this behaviour, I would suggest "memoising" the function (i.e. storing requested data somewhere, and retrieving it if the function is called and the data is recent enough). You will want to use the `useState` hook instead.

(In our case, there's no serious need to have "live updates", since the data change at a slow rate.)